README.md

```diff
@@ -24,31 +24,53 @@ The applications are organized in folders with each of them containing details o
 
 ## Installation
 
-Each application contains detailed instructions on how to build and install the application locally or on a DC/OS cluster or Kubernetes cluster (forthcoming; under development now). See the individual READMEs and `bin` directories for each application for details.
+Each application contains detailed instructions on how to build and install the application locally or on a Fast Data Platform cluster, either on DC/OS, using the `dcos` CLI, or on Kubernetes or OpenShift, using [Helm](https://helm.sh/). See the individual READMEs for each application for details.
 
-Here's how to install any or all of them from a centralized command line script into a DC/OS cluster, using a script `apps/bin/app-install.sh`. This script assumes the following prerequisites:
+The following components must be installed and running on the cluster in advance (See the [Fast Data Platform documentation](https://developer.lightbend.com/docs/fast-data-platform/current/ for details):
 
-* A DC/OS cluster is up and running
+* Kafka
+* Spark
+* Cassandra (required only for KillrWeather app)
+* InfluxDB (required only for KillrWeather and network intrusion apps)
+* Grafana (required only for KillrWeather and network intrusion apps)
+
+### Kubernetes and OpenShift
+
+For both platforms, [Helm](https://helm.sh/) is used to install the sample apps. Make sure the following prerequisites are installed first:
+
+* The Fast Data Platform cluster is up and running
+* The user is authenticated to the cluster
+* The cluster has sufficient resources to host the applications
+* The prerequisites mentioned above are installed.
+
```

Comment:
We do provide helm chart for all prerequisites (except for Spark - no initial install is required) here https://github.com/lightbend/fdp-sample-applications/tree/openshift/supportingcharts - for some reason it is not merged